src/main/java/de/retest/web/selenium/TestHealer.java

```diff
@@ -35,6 +43,7 @@
 	private final UnbreakableDriver wrapped;
 	private final RootElement lastExpectedState;
 	private final RootElement lastActualState;
+	private static ActionReplayResult lastActionReplayResult;
```

Comment:
Why a _static_ mutable field? In [line 55](https://github.com/retest/recheck-web/pull/251/files#diff-2b1201bc248da98862b85ba88ea22564R55), it's not a static invocation.